clef/clef.go

```diff
@@ -29,7 +29,7 @@ type ClefDtctr struct {
 	resultsRlfd2 chan bool
 }
 
-func NewClefDtctr(linkCap float64, alpha, beta_th, beta, gamma uint32, t_l time.Duration) *ClefDtctr {
+func NewClefDtctr(linkCap float64, alpha, beta_th, beta uint32, gamma, t_l time.Duration) *ClefDtctr {
```

Comment:
The current CLEF constructor is not correct, and I am trying to fix it after we make RLFD run correctly. Because we should config EARDet according to given beta_l, gamma_l, and gamma_h (see the NewConfigedEardetDtctr in eardet.go, which was added by me today), so the current CLEF constructor is not correct.

I think maybe a better way is to pass one eardet and two RLFDs as the parameters for CLEF constructor. If you have a chance to fix this, that would be nice, otherwise I will take care next weekend.